### ISSUE TITLE
" TypeError: i.get(...).isEmpty is not a function" at "about/index.jsx"

### ISSUE THREAD

**Issue by fbtw** [mastodon/mastodon#26947]
### Steps to reproduce the problem

1. I upgraded the instance to the version 4.2.0-rc1
2. The main site loads but shows a 501 error "isEmpty is not a function" in the web console

### Expected behaviour

open the home site

### Actual behaviour

server error

### Detailed description

I upgraded the instance to the rc-1 and got a error that prevented the services to load (mastodon-streaming) the error was that ioredis was missing, that error was fixed by reinstalling nodeJS, deleting node_modules and re running "yarn install"  when I finally get it to work the site loads but shows this error message:

```
TypeError: i.get(...).isEmpty is not a function
isEmpty (webpack:///app/javascript/mastodon/features/about/index.jsx:164:52)
wi (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:3592:9)
di (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:3577:10)
Bc (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:1068:2)
bd (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5339:57)
bd (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5336:6)
Error(p (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5325:19)
Xj (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5115:84)
lc.onCommitFiberUnmount (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5136:31)
fa (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:1927:89)
d.hasOwnProperty (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:4519:386)
```

the same message as [this other issue]( https://github.com/mastodon/documentation/issues/1304) from today

Reinstalling yarn, ruby, node, npm and all packages doesn't change the problem, the error is the same after

### Mastodon instance

laburo.social

### Mastodon version

v4.2.0-rc1

### Browser name and version

firefox and chromium 

### Operating system

debian 9

**Comment by ClearlyClaire** [comment]
> 2\. The main site loads but shows a 501 error

What you are describing is a client-side error; are you also seeing a 501 error? In which case, which query is it in response to, and can you find anything in your server logs?

**Comment by fbtw** [comment]
> > 2. The main site loads but shows a 501 error
> 
> What you are describing is a client-side error; are you also seeing a 501 error? In which case, which query is it in response to, and can you find anything in your server logs?

No, it was a error regarding dependencies, it was fixed upgrading to version 4.2.0-rc2
